golangci.yaml

```diff
@@ -34,9 +34,11 @@ linters-settings:
 
 
 issues:
+  exclude-use-default: false
```

Comment:
golangci lint overrides the default settings of revive and other sub-linters to be less pedantic. This setting is saying "don't override the defaults" or "don't exclude the default linter settings" and that's what turns back on godoc linting (among a few others, see earlier commits in this PR where the linters fail for full set of examples in our codebase)